instrumentation/excon/lib/opentelemetry/instrumentation/excon/middlewares/tracer_middleware.rb

```diff
@@ -10,25 +10,16 @@ module Excon
       module Middlewares
         # Excon middleware for instrumentation
         class TracerMiddleware < ::Excon::Middleware::Base
-          HTTP_METHODS_SYMBOL_TO_STRING = {
-            connect: 'CONNECT',
-            delete: 'DELETE',
-            get: 'GET',
-            head: 'HEAD',
-            options: 'OPTIONS',
-            patch: 'PATCH',
-            post: 'POST',
-            put: 'PUT',
-            trace: 'TRACE'
-          }.freeze
+          HTTP_METHODS_TO_UPPERCASE = Hash.new { |hash, key| hash[key] = key.to_s.upcase }
+          HTTP_METHODS_TO_SPAN_NAMES = Hash.new { |hash, key| hash[key] = "HTTP #{key}" }
```

Comment:
This hash should also be immutable. It should probably be based on the values from the `HTTP_METHODS_TO_UPPER_CASE`.